### PR TITLE
fix(cheatsheets/python): fix typos and inconsistencies

### PR DESCRIPTION
**Pull request by tclain**
I had missed some dummy typos 😅 

It should be better now.

### CODE REVIEW

**Comment by tclain** [comment]
@lpil  if you want to approve this. thanks !

File: cheatsheets/gleam-for-python-users.md [hunk 1]
```diff
@@ -133,7 +184,7 @@ These annotations are accessible at runtime via the `__annotations__` module-lev
 These hints will mainly be used to inform static analysis tools like IDEs, linters...
 
 ```py
-from typing import List, int
```
Comment (tclain) [inline cheatsheets/gleam-for-python-users.md:136]: int is already in scope

File: cheatsheets/gleam-for-python-users.md [hunk 2]
```diff
@@ -4,33 +4,84 @@ title: Gleam for Python users
 ---
 
 - [Comments](#comments)
+    - [Python](#python)
+    - [Gleam](#gleam)
```
Comment (lpil) [inline cheatsheets/gleam-for-python-users.md:8]: Could you remove these extra contents links that say "Python" and "Gleam" please? Thank you

Comment (tclain) [reply cheatsheets/gleam-for-python-users.md:8]: It's my markdown plug-in that generate the TOC 🤔, will remove them

**Review by lpil — APPROVED** [review]
Thank you

**Pull request merged**